pkgs/tools/security/hashcash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "hashcash-${version}";
```

Comment:
You can just use `pname = "hashcash"` nowadays.

Reply:
I didn't know this.  Thanks!